.github/ISSUE_TEMPLATE/support-request.yml

```diff
@@ -1,6 +1,9 @@
 name: 🤖 HW Support Request
+title: Short Failure Description
 description: Support request for HW maintenance
 assignees: "sgiraz,AntonioConsilvio"
+labels: ["form-type-support"]
```

Comment:
This issue label is not available yet in the set.

At any rate, I've been using it in our purchase system only to differentiate among issue types (we have therein different types of orders). Since here we have only one type, we may not need it.

Reply:
I just removed the label as it isn't necessary anymore.